spec/features/parental_consent_refused_spec.rb

```diff
@@ -90,4 +99,57 @@ def then_i_see_the_consent_page
   def then_i_can_check_my_answers
     expect(page).to have_content("Check your answers")
   end
+
+  def when_i_confirm_my_answers
+    click_on "Confirm"
+  end
+
+  def then_i_see_the_confirmation_page
+    expect(page).to have_content(
+      "Your child will not get an HPV vaccination at school"
+    )
+  end
+
+  def and_i_receive_an_email_confirming_that_my_child_wont_be_vaccinated
+    expect(enqueued_jobs.first["scheduled_at"]).to be_nil
+    expect(
+      Time.zone.parse(enqueued_jobs.second["scheduled_at"]).to_i
+    ).to be_within(1.second).of(1.hour.from_now.to_i)
+
+    expect_email_to "jane@example.com",
```

Comment:
After finding that too many emails were being sent out as part of the triage needed vaccination journey, I'd be tempted to add the expectation on the number of emails being sent. We don't test that part of the journeys anywhere else.